compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/DatePicker.kt

```diff
@@ -1798,24 +1799,15 @@ private fun MonthsNavigation(
         // Show arrows for traversing months (only visible when the year selection is off)
         if (!yearPickerVisible) {
             Row {
-                val rtl = LocalLayoutDirection.current == LayoutDirection.Rtl
                 IconButton(onClick = onPreviousClicked, enabled = previousAvailable) {
                     Icon(
-                        if (rtl) {
-                            Icons.Filled.KeyboardArrowRight
-                        } else {
-                            Icons.Filled.KeyboardArrowLeft
-                        },
+                        imageVector = Icons.AutoMirrored.Filled.KeyboardArrowLeft,
                         contentDescription = getString(Strings.DatePickerSwitchToPreviousMonth)
                     )
                 }
                 IconButton(onClick = onNextClicked, enabled = nextAvailable) {
                     Icon(
-                        if (rtl) {
-                            Icons.Filled.KeyboardArrowLeft
-                        } else {
-                            Icons.Filled.KeyboardArrowRight
-                        },
+                        imageVector = Icons.AutoMirrored.Filled.KeyboardArrowRight,
```

Comment:
Does it work on iOS?
See https://github.com/JetBrains/compose-multiplatform/issues/4172

Reply:
It worked for me in `widgets-gallery`.

<img width="58" alt="image" src="https://github.com/JetBrains/compose-multiplatform-core/assets/5230206/5fd7ae78-3e80-4ee9-8e9c-37e3b844b4fb">